frontend/public/components/app.jsx

```diff
@@ -92,7 +92,7 @@ class App extends React.PureComponent {
     // Some components, like svg charts, need to reflow when nav is toggled.
     // Fire event after a short delay to allow nav animation to complete.
     setTimeout(() => {
-      window.dispatchEvent(new Event('nav_toggle'));
+      window.dispatchEvent(new Event('component_resize_toggle'));
```

Comment:
Maybe `sidebar_toggle`? Nav and drawer both seem like sidebars.

We should move this to const.ts as well since it's in several places now.